src/Acquisition/_Acquisition.c

```diff
@@ -2030,7 +2109,22 @@ init_Acquisition(void)
   AcquisitionCAPI.AQ_Inner = capi_aq_inner;
   AcquisitionCAPI.AQ_Chain = capi_aq_chain;
 
-  api = PyCObject_FromVoidPtr(&AcquisitionCAPI, NULL);
+  api = PyCapsule_New(&AcquisitionCAPI, "AcquisitionCAPI", NULL);
```

Comment:
Is the `AcquisitionCAPI` name correct here? Or does this need to be a dotted name to a module now? The C extension is built as `Acquisition._Acquisition`.

Acquisition.h also uses this via `PyCapsule_Import("Acquisition.AcquisitionCAPI", 0)`.